### PR TITLE
git action

### PR DESCRIPTION
**Pull request by baiyutang**
1. add git action config.
2. remove example dir，the code writed in [kitex-examples](https://github.com/cloudwego/kitex-examples)  is better.
3. update  copyright.

### CODE REVIEW

**Comment by lizhemingi** [comment]
Thank you for your contribution.

I think keep the example would be better, since we can easily find a usage example in the current repo and run when cloned instead of going to another one.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (lizhemingi) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: Maybe Desktop and Smartphone is not necessary ?

Comment (lizhemingi) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: @baiyutang I am sorry but it seems that this one is marked as resolved but not changed.

**Comment by lizhemingi** [comment]
It would be very nice if you can squash the commits after you finished the code. @baiyutang

**Comment by baiyutang** [comment]
> It would be very nice if you can squash the commits after you finished the code. @baiyutang

done

**Comment by GuangmingLuo** [comment]
> 1. add git action config.
> 2. remove example dir，the code writed in [kitex-examples](https://github.com/cloudwego/kitex-examples)  is better.
> 3. update  copyright.

you may update your pr description.

**Review by lizhemingi — APPROVED** [review]

**Pull request merged**